client/webpack.client.rails.build.config.js

```diff
@@ -69,6 +69,9 @@ if (devBuild) {
   console.log('Webpack dev build for Rails'); // eslint-disable-line no-console
   config.devtool = 'eval-source-map';
 } else {
+  config.plugins.push(
+    new webpack.optimize.UglifyJsPlugin()
+  );
```

Comment:
@lukeasrodgers Can you check if this is really needed. Since we're loading the client file through the asset pipeline, it should be minified.

**HOWEVER** the server side file might benefit from this plugin, since it does not go through the asset pipeline.

@alexfedoseev, thoughts?